Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml

```
@@ -261,11 +265,20 @@
   - type: SolutionScanner
+  - type: CPRTraining
+  - type: SurgerySpeedModifier
+    speedModifier: 1.75
   - type: InteractionPopup
     interactSuccessString: petting-success-medical-cyborg
     interactFailureString: petting-failure-medical-cyborg
     interactSuccessSound:
       path: /Audio/Ambience/Objects/periodic_beep.ogg
+  - type: ShowHealthBars
+    damageContainers:
+    - Biological
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)